docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-network-connection-via-registration-utility.asciidoc

```diff
@@ -0,0 +1,84 @@
+[[prebuilt-rule-0-16-2-network-connection-via-registration-utility]]
+=== Network Connection via Registration Utility
+
+Identifies the native Windows tools regsvr32.exe, regsvr64.exe, RegSvcs.exe, or RegAsm.exe making a network connection. This may be indicative of an attacker bypassing allowlists or running arbitrary scripts via a signed Microsoft binary.
```

Comment:
```suggestion
Identifies the native Windows tools `regsvr32.exe`, `regsvr64.exe`, `RegSvcs.exe`, or `RegAsm.exe` making a network connection. This may be indicative of an attacker bypassing allowlists or running arbitrary scripts via a signed Microsoft binary.
```